features/app/routerMiddleware.web.js

```diff
@@ -0,0 +1,25 @@
+import { push } from 'react-router-redux';
+
+import { APP_NAVIGATE } from './actionTypes';
+import { APP_SCREEN } from './constants';
+
+/**
+ * This router middleware is used for navigation inside the web app.
+ *
+ * @param {Store} store - Redux store.
+ * @returns {Function}
+ */
+export function routerMiddleware(store) {
+    return next => action => {
+        if (action.type === APP_NAVIGATE) {
+            switch (action.screen) {
+            case APP_SCREEN.WELCOME:
+                return store.dispatch(push('/'));
```

Comment:
We talked about it in an online meeting. I'm just putting our that talk here in its context:

routerMiddleware appears to be suspiciously similar between .native.js and .web.js so I'd like us to explore the possibility to expand the abstraction in the direction of having 1 platform-independent routerMiddleware and other smaller platform-dependent functions that it calls.